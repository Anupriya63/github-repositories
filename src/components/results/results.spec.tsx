import React from 'react';
import { shallow } from 'enzyme';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store'; 
import PageList from './list';

const initialState = {
    repos: [],
    error: '',
    notFound: false,
};

const mockStore = configureStore();
const defaultStore = mockStore(initialState);


describe('Results', () => {

    it('render results if repository is greater than 0', () => {
        const pageList = shallow(
            <Provider store={defaultStore}>
                <PageList />
            </Provider>);
        expect(pageList).toMatchSnapshot();
    });
});
