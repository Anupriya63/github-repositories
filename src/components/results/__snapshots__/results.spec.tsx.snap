// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Results render results if repository is greater than 0 1`] = `ShallowWrapper {}`;
